lib/writer.js

```diff
@@ -128,6 +129,27 @@ class Writer {
     });
   }
 
+  /**
+   * Check whether custom template path is specified.
+   *
+   * @returns {boolean}
+   */
+  isUsingDefaultTemplate () {
+    const defaultTemplateDir = this.path.join(__dirname, '..', 'template') + this.path.sep;
+    return this.opt.template === defaultTemplateDir;
+  }
+
+  /**
+   * Get the path of webpack.config.js
+   *
+   * @returns {string}
+   */
+  getWebpackConfigPath () {
+    return this.isUsingDefaultTemplate()
+      ? this.path.join(this.opt.template, 'src', 'webpack.config.js')
+      : this.path.join(__dirname, '/../../../', this.opt.template, 'src', 'webpack.config.js');
```

Comment:
I don't think this is the correct approach. `this.opt.template` must be absolute path and that's it. Here the webpack config is at the same place than in the default template, so the issue is more about the template path rather than the webpack config path.